docs/guides/deploy_service.md

```diff
@@ -40,7 +42,7 @@ Local deployments of a service are recommended to test your service before you p
 
 2. Build the Docker image of the service agents:
     ```bash
-    autonomy build-image
+    autonomy build-image --dev
```

Comment:
@angrybayblade Please indicate if we need to download packasges at this point.

Reply:
Yes it will be an overkill for this tutorial but we can optimise it laters, right now we need to make sure that the user knows that they need the packages directory and all the components present if they want to run a local deployment